scripts/sources/parameters/cmd/aws_ssm.go

```diff
@@ -0,0 +1,58 @@
+/*
+Based on: https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/gov2/ssm/GetParameter/GetParameterv2.go
+*/
+package parameters
+
+import (
+	"context"
+	"log"
+
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+)
+
+// SSMGetParametersAPI and SSMDescribeParametersAPI defines the interface
+// for the GetParameters and DescribeParameters function.
+// We use this interface to test the function using a mocked service
+type SSMGetParametersAPI interface {
```

Comment:
Do we need to have two different interfaces? Are we going to use them separately? If not, having two interfaces makes the code unnecessarily complicated.